src/OrchardCore.Modules/OrchardCore.Features/AdminMenu.cs

```diff
@@ -22,7 +22,7 @@ public Task BuildNavigationAsync(string name, NavigationBuilder builder)
             }
 
             builder
-                .Add(T["Configuration"], "4", configuration => configuration
+                .Add(T["Configuration"], "9", configuration => configuration
                     .AddClass("menu-configuration").Id("configuration")
```

Comment:
Nice one @agriffard 

One last thing I notice, this could be 100 everywhere, so that it always stays at the bottom. 
Even when we add new menu items.
Other places it is 10, here it is 9. Or maybe it just needs to be 100 once, somewhere on a feature that is always on. and never specified again.

Or maybe it should be set on `Settings` the once. That would always be on? I think

Reply:
ok @deanmarcussen Configuration = "100"

Sorry, I didn't remember I switched back my branch to dev so I already pushed it to dev but I guess it doesn't harm.